internal/services/cockpit/alert_manager.go

```diff
@@ -0,0 +1,260 @@
+package cockpit
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/scaleway/scaleway-sdk-go/api/cockpit/v1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"github.com/scaleway/terraform-provider-scaleway/v2/internal/services/account"
+	"github.com/scaleway/terraform-provider-scaleway/v2/internal/verify"
+)
+
+func ResourceCockpitAlertManager() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: ResourceCockpitAlertManagerCreate,
+		ReadContext:   ResourceCockpitAlertManagerRead,
+		UpdateContext: ResourceCockpitAlertManagerUpdate,
+		DeleteContext: ResourceCockpitAlertManagerDelete,
+		Timeouts: &schema.ResourceTimeout{
+			Create:  schema.DefaultTimeout(DefaultCockpitTimeout),
+			Read:    schema.DefaultTimeout(DefaultCockpitTimeout),
+			Delete:  schema.DefaultTimeout(DefaultCockpitTimeout),
+			Default: schema.DefaultTimeout(DefaultCockpitTimeout),
+		},
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Schema: map[string]*schema.Schema{
+			"project_id": account.ProjectIDSchema(),
+			"enable": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"emails": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString, ValidateFunc: verify.IsEmail()},
+				Optional:    true,
+				Description: "A list of email addresses for the alert receivers",
+			},
+		},
+	}
+}
+
+func ResourceCockpitAlertManagerCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	api, region, err := cockpitAPIWithRegion(d, meta)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	projectID := d.Get("project_id").(string)
+	enable := d.Get("enable").(bool)
+	emails := d.Get("emails").([]interface{})
+
+	_, err = api.EnableAlertManager(&cockpit.RegionalAPIEnableAlertManagerRequest{
+		ProjectID: projectID,
```

Comment:
The alert manager is [regional](https://github.com/scaleway/scaleway-sdk-go/blob/master/api/cockpit/v1/cockpit_sdk.go#L954), please add the Region field